torchx/workspace/test/docker_workspace_test.py

```diff
@@ -52,7 +52,7 @@ def test_docker_workspace(self) -> None:
 
             workspace = DockerWorkspaceMixin()
             workspace.build_workspace_and_update_role(
-                role, "memory://test_workspace", {}
+                role, "memory://test_workspace", {"image_repo": "example/repo_name"}
```

Comment:
To fix the python unittest. This sets cfg["image_repo"] to "example/repo_name", since the image is not pushed during the test case, it will not cause any problem.